src/controller_examples/zookeeper_controller/trusted/zookeeper_api_model.rs

```diff
@@ -5,8 +5,8 @@ use crate::kubernetes_api_objects::{
     common::*, config_map::*, dynamic::*, resource::*, stateful_set::*,
 };
 use crate::vstd_ext::string_view::*;
-use crate::zookeeper_controller::common::*;
-use crate::zookeeper_controller::spec::{types::ZookeeperClusterView, resource::make_zk_config};
+use crate::zookeeper_controller::model::resource::make_zk_config;
```

Comment:
The trusted code should not import from the model, so we should move the code to the trusted component.

Reply:
I think the structure of `trusted` is a bit rough. Should I just put in in zk_api_spec?

Reply:
Can you create a new file `trusted/config_data_serialization.rs` and put it there?

Reply:
Just edited --- let's just call it `configmap.rs`